apps/site/vercel-lib.sh

```diff
@@ -1,7 +1,7 @@
 # Install python3-saml dependencies on Vercel's Serverless Function 
 # environment (Amazon Linux)
 
-yum install -y libxml2-devel xmlsec1-devel xmlsec1-openssl-devel libtool-ltdl-devel
+yum install -y libxml2-devel xmlsec1-devel xmlsec1-openssl-devel libtool-ltdl-devel libxslt-devel libgcrypt-devel libgpg-error-devel
```

Comment:
Thought: the extra libraries added should already be transitive dependencies of the ones we need, so I don't think we need to manually specify them.

Reply:
Chore [partly out of scope]: while you're at it, I think `libtool-ltdl-devel` is also a transitive dependency and can be removed.